tensorflow/python/keras/layers/advanced_activations.py

```diff
@@ -306,6 +306,13 @@ class ReLU(Layer):
 
   def __init__(self, max_value=None, negative_slope=0, threshold=0, **kwargs):
     super(ReLU, self).__init__(**kwargs)
+    if isinstance(max_value, dict):
```

Comment:
Just do the same fix as Keras
```suggestion
   if max_value is not None:
       max_value = K.cast_to_floatx(max_value)
```

Reply:
Thank you so much for responding. I assume that the bug was due to `tf.keras.models.load_model`. It loads layer information from `*.h5`. The argument of Relu layer like `max_value=None`, it loads None. But if the argument is a number(like `negative_slope=0`), it loads a dict. So the error is caused by `if max_value is not None and max_value < 0` or `if negative_slope < 0`.
And the bug you mentioned above, it solved the problem that if we cast `None` to `K.cast_to_floatx(max_value)` would return `nan`.